OracleOpenJDK/14/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle OpenJDK 14
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+#
+# (1) openjdk-14.xx_linux-x64_bin.tar.gz
+#     Download from https://jdk.java.net/14/
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -t oracle/openjdk:14 .
+#
+# This command is already scripted on build.sh so you can alternatively run
+#		$ bash build.sh
+#
+# The builder image will be use to uncompress the tar.gz file with the Java Runtime.
+
+FROM oraclelinux:7-slim as builder
+
+MAINTAINER Aurelio Garcia-Ribeyro <aurelio.garciaribeyro@oracle.com>
+
+# Since the files is compressed as tar.gz first yum install gzip and tar
+RUN set -eux; \
+	yum install -y \
+		gzip \
+		tar \
+	; \
+	rm -rf /var/cache/yum
+	
+
+# Default to UTF-8 file.encoding
+ENV LANG en_US.UTF-8
+
+# Environment variables for the builder image.  
+# Required to validate that you are using the correct file
+
+ENV JAVA_PKG=openjdk-14.0.2_linux-x64_bin.tar.gz \
+	JAVA_SHA256=91310200f072045dc6cef2c8c23e7e6387b37c46e9de49623ce0fa461a24623d \
+	JAVA_HOME=/usr/java/jdk-14
+
+COPY $JAVA_PKG /tmp/jdk.tgz
```

Comment:
Instead of this `COPY` directive, add a `curl` command to the followiung `RUN` directive to download the package during build.